src/components/hint/index.tsx

```diff
@@ -217,15 +217,13 @@ class Hint extends Component<HintProps, HintState> {
     if (!_.isEqual(this.state.targetLayout, layout)) {
       this.setState({targetLayout: layout});
     }
-
-    if (!this.state.targetLayoutInWindow) {
-      setTimeout(() => {
-        this.targetRef?.measureInWindow((x: number, y: number, width: number, height: number) => {
-          const targetLayoutInWindow = {x, y, width, height};
-          this.setState({targetLayoutInWindow});
-        });
+    
+    setTimeout(() => {
```

Comment:
Maybe it's make sense to re-measure only if the onBackgroundPress prop has been sent, cause otherwise it's not necessary

Reply:
Done